io/prometheus/client/metrics.proto

```diff
@@ -78,26 +80,27 @@ message Histogram {
   // then each power of two is divided into 2^n logarithmic buckets.
   // Or in other words, each bucket boundary is the previous boundary times 2^(2^-n).
   // In the future, more bucket schemas may be added using numbers < -4 or > 8.
-  optional sint32 schema           = 5;
-  optional double zero_threshold   = 6; // Breadth of the zero bucket.
-  optional uint64 zero_count       = 7; // Count in zero bucket.
-  optional double zero_count_float = 8; // Overrides sb_zero_count if > 0.
+  optional sint32                    schema             = 5;
+  optional double                    zero_threshold     = 6; // Breadth of the zero bucket.
+  optional uint64                    zero_count         = 7; // Count in zero bucket.
+  optional double                    zero_count_float   = 8; // Overrides sb_zero_count if > 0.
 
   // Negative buckets for the native histogram.
-  repeated BucketSpan negative_span =  9;
+  repeated BucketSpan                negative_span      =  9;
   // Use either "negative_delta" or "negative_count", the former for
   // regular histograms with integer counts, the latter for float
   // histograms.
-  repeated sint64 negative_delta    = 10; // Count delta of each bucket compared to previous one (or to zero for 1st bucket).
-  repeated double negative_count    = 11; // Absolute count of each bucket.
+  repeated sint64                    negative_delta     = 10; // Count delta of each bucket compared to previous one (or to zero for 1st bucket).
+  repeated double                    negative_count     = 11; // Absolute count of each bucket.
 
   // Positive buckets for the native histogram.
-  repeated BucketSpan positive_span = 12;
+  repeated BucketSpan                positive_span      = 12;
   // Use either "positive_delta" or "positive_count", the former for
   // regular histograms with integer counts, the latter for float
   // histograms.
-  repeated sint64 positive_delta    = 13; // Count delta of each bucket compared to previous one (or to zero for 1st bucket).
-  repeated double positive_count    = 14; // Absolute count of each bucket.
+  repeated sint64                     positive_delta    = 13; // Count delta of each bucket compared to previous one (or to zero for 1st bucket).
+  repeated double                     positive_count    = 14; // Absolute count of each bucket.
+  optional google.protobuf.Timestamp  created_timestamp = 15;
```

Comment:
Let's separate this by a blank line and a comment explaining that it doesn't belong to native histograms specifically anymore (and also update the comment in line 75, which currently reads "Everything below here is for native histograms…").

Reply:
Maybe you could even pull it up to add this line just after the classic buckets. Might be least confusing, even if the field numbering is out of order then.

Reply:
I just moved it upwards :)